virt/virt-4-10-release-notes.adoc

```diff
@@ -0,0 +1,228 @@
+
+[id="virt-4-10-installation-new"]
+=== Installation
+
+//CNV-14192 Update launchers using live-migration - default/opt out
+
+//CNV-14910 HyperConverged Operator technical debt
+
+[id="virt-4-10-networking-new"]
+=== Networking
+
+//CNV-11201 Rollout of NodeNetworkConfigurationPolicy
+
+//CNV-13660 Inherit static IP from a NIC attached to the bridge
+
+//CNV-13679 SR-IOV live-migration without privilege escalation
+
+//CNV-13680 Primary and secondary networks over a shared NIC on OVN clusters
+
+//CNV-8577 CNV-14481 Service Mesh for VMs on the primary network with IPv4 (Done in CNV-8577 for 4.9 then reverted in CNV-14481)
+
+[id="virt-4-10-storage-new"]
+=== Storage
+
+[id="virt-4-10-web-new"]
+=== Web console
+
+// NOTE: no deprecated features in 4.10; commenting out these sections to retain for future use
+//[id="virt-4-10-deprecated-removed"]
+//== Deprecated and removed features
+
+//[id="virt-4-10-deprecated"]
+//=== Deprecated features
+
+//Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
+
+// NOTE: when uncommenting deprecated features list, change the header level below to ===
+
+//[id="virt-4-10-removed"]
+//=== Removed features
+
+//Removed features are not supported in the current release.
+
+// NOTE: no notable technical changes in 4.9, commenting out to retain
+//[id="virt-4-10-changes"]
+//== Notable technical changes
+
+[id="virt-4-10-technology-preview"]
+== Technology Preview features
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+//CNV-14192 virt-launcher now automatically updates - now TP
+* You can now enable automatic updates for {VirtProductName} workloads, such as `virt-launcher` pods. xref:../virt/upgrading-virt.adoc#configuring-workload-updates_upgrading-virt[Configure workload update strategies] by editing the `HyperConverged` custom resource.
+
+* You can now xref:../virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc#virt-hot-plugging-virtual-disks[hot-plug and hot-unplug virtual disks] when you want to add or remove them from your virtual machine without stopping the virtual machine instance.
+
+* You can now use the Red Hat Enterprise Linux 9 Alpha template to create virtual machines.
+
+* You can now link:https://access.redhat.com/articles/6409731[deploy {VirtProductName} on AWS bare metal nodes].
+
+[id="virt-4-10-bug-fixes"]
+== Bug fixes
+
```

Comment:
I assume that all bug fixes from 4.9 can be removed from the 4.10 RNs

Reply:
Gotcha, done.